api/models/timelog.py

```diff
@@ -70,7 +70,7 @@ class Template(Base):
     customer_id = Column("customerid", Integer, ForeignKey("customer.id"), nullable=True)
     user_id = Column("usrid", Integer, ForeignKey("usr.id"), nullable=True)
     project_id = Column("projectid", Integer, ForeignKey("project.id"), nullable=True)
-    is_global = Column(Boolean, nullable=False)
+    is_global = Column(Boolean, nullable=False, default=False)
```

Comment:
Sorry for the nit, but I think this belongs to the previous commit.

Reply:
Never be sorry for nits ;) Should be fixed now